src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -154,7 +149,7 @@ private[scoverage] abstract class ScoverageSensorInternal extends Sensor {
   }
 
   /** Saves the [[ScoverageMetrics]] of a component */
-  private[this] def saveComponentScoverage(
```

Comment:
Just the same about the logger.
It's there any reason for not using `private[this]`?
Maybe a performance issue?

IMHO, one should be as private as possible -  exposing only the necessary.
But, I've learn that scala do funny thnigs sometimes when compiling.

Reply:
I changed the scope so I could unit test this in isolation.

Reply:
Oh ok :+1: